pages/components/Collection/Collection.js

```diff
@@ -1,5 +1,69 @@
-const Collection = () => (
-  <h1>123</h1>
-)
+import S from "@/pages/components/Collection/Collection.module.scss"
+import React, { useState } from "react"
+import Image from "next/image"
+import prevItemImage from "../../../public/prevItem.png"
+import nextItemImage from "../../../public/nextItem.png"
+import { wines, wineType } from "@/pages/utils/varibles"
+import WineButton from "@/pages/components/WineButton"
+import CollectionItem from "@/pages/components/CollectionItem"
+import WineCollectionCircles from "@/pages/components/WineCollectionCircles"
+
+const Collection = () => {
+  const [ winesCollection, setWinesCollection ] = useState(wines)
+  const [ currentIndex, setCurrentIndex ] = useState(0)
+  const [ activeButton, setActiveButton ] = useState(null)
+
+  const sort = (event) => {
+    setCurrentIndex(0)
+    setWinesCollection(wines.filter((element) => {
+      return element.classification === event
+    }))
+  }
+
+  const changeColor = (index) => {
+    setActiveButton(index)
+  }
+
+  const nextItem = () => {
+    setCurrentIndex((prevIndex) => (prevIndex + 1) % winesCollection.length)
+  }
```

Comment:
```js
  const nextItem = () => {
    setCurrentIndex((prevIndex) => (prevIndex + 1) % winesCollection.length)
  }
```

```js
  const nextItem = () => setCurrentIndex(prevIndex => (prevIndex + 1) % winesCollection.length)
```

I would like to know your setting of the linter.
Some things could be written in a shorter way